runtime/plugin/difftool.lua

```diff
@@ -0,0 +1,12 @@
+if vim.g.loaded_difftool_plugin ~= nil then
+  return
+end
+vim.g.loaded_difftool_plugin = true
+
+vim.api.nvim_create_user_command('DiffTool', function(opts)
+  if #opts.fargs == 2 then
+    require('difftool').diff(opts.fargs[1], opts.fargs[2])
```

Comment:
Move `runtime/lua/difftool.lua` to `runtime/lua/nvim/difftool.lua` , that's where plugins live. This is not part of the stdlib.

```suggestion
    require('nvim.difftool').diff(opts.fargs[1], opts.fargs[2])
```